ompi/mca/mtl/ofi/mtl_ofi.h

```diff
@@ -999,6 +999,67 @@ ompi_mtl_ofi_send_generic(struct mca_mtl_base_module_t *mtl,
     return ofi_req.status.MPI_ERROR;
 }
 
```

Comment:
There should be a small comment header describing what this function does, when to call it, any non obvious assumptions.